src/cli/commands/gsn-relayer-register.ts

```diff
@@ -5,16 +5,17 @@ import { configureGSN } from '../../relayclient/GSNConfigurator'
 import { getNetworkUrl, getRelayHubAddress, gsnCommander, getMnemonic } from '../utils'
 
 const commander = gsnCommander(['n', 'f', 'h', 'm'])
-  .option('--relayUrl <url>', 'url to advertise the relayer (defaults to localhost:8090)')
-  .option('--stake <stake>', 'amount to stake for the relayer, in wei (defaults to 1 Ether)')
+  .option('--relayUrl <url>', 'url to advertise the relayer', 'http://localhost:8090')
+  .option('--stake <stake>', 'amount to stake for the relayer, in ETH', '1')
   .option(
     '--unstakeDelay <delay>',
-    'blocks to wait between unregistering and withdrawing the stake (defaults to one 1000)'
+    'blocks to wait between unregistering and withdrawing the stake', '1000'
   )
   .option(
     '--funds <funds>',
-    'amount to transfer to the relayer to pay for relayed transactions, in wei (defaults to 2 Ether)'
+    'amount to transfer to the relayer to pay for relayed transactions, in ETH', '2'
   )
+  .option('-g, --gasPrice <number>', 'gas price to give to the transaction, in Gwei. Defaults to relay\'s minGasPrice')
```

Comment:
you don't need to manually add it. Just pass 'g' to the `gsnCommander` constructor